modules/vulnerability-management20-retention-settings.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-vulnerabilities.adoc
+:_content-type: PROCEDURE
+[id="vulnerability-management20-retention-settings_{context}"]
+= Configuring vulnerability management report job retention settings
+
+[role="_abstract"]
+You can configure settings that determine when vulnerability report job requests expire and other retention settings for report jobs.
+
+These settings do not affect the following vulnerability report jobs:
+
+* Jobs in the `WAITING` or `PREPARING` state
+* The last successful scheduled report job
+* The last successful on-demand emailed report job
+* The last successful downloadable report job
+* Downloadable report jobs for which the report file has not been deleted by either manual deletion or by configuring the downloadable report pruning settings
+
+.Procedure
+. In the {product-title-short} web portal, navigate to *Platform Configuration* -> *System Configuration*. You can configure the following settings for vulnerability report jobs:
+* *Expired vulnerability requests*: The number of days after which a vulnerability report job request expires. The default value is 90 days. ##Is there a maximum and minimum value?##
+* *Vulnerability report run history retention*: The number of days that a record is kept of vulnerability report jobs that have been run. This setting controls how many days that report jobs are listed in the *All report jobs* tab under *Vulnerability Management 2.0* -> *Vulnerability Reporting* when a report configuration is selected. ##Default, min, and max values?##
```

Comment:
There are a few exceptions that should be called out:
The report history garbage collector prunes all report history that is beyond cutoff except:

- Unfinished jobs
- Jobs for which prepared downloadable reports still exist in the system.
 - Last successful report job for each job type—scheduled email, on-demand email, or download. This ensures users have information about the last run for each type.

Reply:
These were already added - line 13 through 17?

Reply:
The above-mentioned exceptions are specific to `Vulnerability report run history retention`